src/main/java/com/metamx/emitter/core/Emitters.java

```diff
@@ -1,107 +0,0 @@
-/*
- * Copyright 2012 Metamarkets Group Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.metamx.emitter.core;
-
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.google.common.collect.Maps;
-import com.metamx.common.IAE;
-import com.metamx.common.ISE;
-import com.metamx.common.lifecycle.Lifecycle;
-import com.metamx.common.logger.Logger;
-import com.metamx.http.client.HttpClient;
-
-import java.util.Map;
-import java.util.Properties;
-
-public class Emitters
```

Comment:
its not needed anymore, it was used by unit tests and was doing unnecessary conversions. pls see the updated unit tests.